templates/web/smidsy/index.html

```diff
@@ -1,6 +1,9 @@
 [% pre_container_extra = PROCESS 'around/postcode_form.html' %]
 [% SET bodyclass = 'frontpage fullwidthpage' %]
 [% INCLUDE 'header.html', title = '' %]
+[% extra_js = [
+    version('/cobrands/smidsy/js/front-page.js'),
```

Comment:
Could this be added to templates/web/smidsy/footer_extra_js.html instead (e.g. this overwrites the reports chart JS loading, which doesn't matter here but extra_js is really for internal extras, I think).